dataland-frontend/src/components/resources/chooseFrameworkForDataUpload/MetaInfoPerCompanyAndFramework.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <div>
+    <div>
+      <h4>{{ title + " Datasets:" }}</h4>
+      <div v-if="isWaitingForData" class="inline-loading text-center">
+        <p class="font-medium text-xl">Loading...</p>
+        <i class="pi pi-spinner pi-spin" aria-hidden="true" style="z-index: 20; color: #e67f3f" />
```

Comment:
On the upload data page: In the searchbar, where one can find companies to add data to, the spinning wheel is grey and appears on the left side of the bar, where it overlaps with the magnifying glass

Reply:
This seems to be the case for all searchbars